### PR TITLE
fix(adt): Updated comments to generate more meaningful API documentation

### PR DESCRIPTION
**Pull request by vinagesh**
https://msazure.visualstudio.com/One/_workitems/edit/7342957
https://msazure.visualstudio.com/One/_workitems/edit/7343224
AB#[7342957]

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs [hunk 1]
```diff
@@ -421,7 +421,7 @@ Page<string> NextPageFunc(string nextLink, int? pageSizeHint)
         }
 
         /// <summary>
-        /// Gets all the relationships referencing a digital twin as a target.
+        /// Gets all the relationships referencing a digital twin as a target asynchronously. After the task completes response would be set to <see cref="AsyncPageable{T}"/>.
```
Comment (abhipsaMisra) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs:424]: This shouldn't really say `After the task completes ...`. Iterating through the `AsyncPageable` is an async operation. We should probably highlight that as well.

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json [hunk 2]
```diff
@@ -0,0 +1,3270 @@
+﻿{
```
Comment (drwill-ms) [inline sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json:1]: This doesn't belong in this PR

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs [hunk 3]
```diff
@@ -96,15 +96,15 @@ public DigitalTwinsClient(Uri endpoint, TokenCredential credential, DigitalTwins
...
+        /// Creates a new instance of the <see cref="DigitalTwinsClient"/> class.
         /// </summary>
         protected DigitalTwinsClient()
         {
             // This constructor only exists for mocking purposes in unit tests. It should not be used otherwise
         }
 
         /// <summary>
-        /// Get a digital twin.
+        /// Gets a digital twin asynchronously. After the task completes response would be set to <see cref="Response{T}"/>.
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs:107]: This reads a bit weird. I'd reword it to be:
> After the task completes, the result will be set to <see cref="Response{T}"/>

**Review by drwill-ms — APPROVED** [review]

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs (hunk 1 above)
Comment (vinagesh) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs:424]: Good catch. Will update all the comments for the pageable responses.

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json (hunk 2 above)
Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json:1]: oops this got included looks like. Will remove it.

**Review by barustum — APPROVED** [review]
